.docs/content/2.usage/use-armonik-grpc-securely-python.md

```diff
@@ -0,0 +1,75 @@
+# Connecting to ArmoniK Securely using gRPC in Python
+
+## Overview
+
+This guide provides steps to connect to ArmoniK securely using gRPC in Python. ArmoniK supports both TLS and mTLS (mutual TLS) for secure communication.
+
+### Prerequisites
+
+Before proceeding, make sure that ArmoniK is deployed with the necessary certificates. Follow the steps below:
+
+1. In the `parameters.tfvars` file of ArmoniK, set the following values in the Deploy Ingress section:
+
+    For TLS only:
+
+    ```hcl
+    tls = true
+    mtls = false
+    ```
+
+    For mTLS:
+
+    ```hcl
+    tls = true
+    mtls = true
+    ```
+
+2. Deploy ArmoniK.
+
+3. Certificates will be generated in `all-in-one/generated/certificates/ingress`:
+   - For TLS: `ca.crt`
+   - For mTLS: `ca.crt`, `client.submitter.crt`, `client.submitter.key`
```

Comment:
ArmoniK.Api doesn't depend on the infrastructure. So I'm unsure if this documentation should be here ? @aneojgurhem what do you think ?